src/test/java/com/thealgorithms/dynamicprogramming/EgsDroppingTest.java

```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.dynamicprogramming;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EgsDroppingTest {
```

Comment:
```suggestion
public class EggDroppingTest {
```

please rename the class and java file

Reply:
Done